common.blocks/link/link.en.md

```diff
@@ -13,6 +13,7 @@ Use this block for creating different types of links.
 | <a href="#focused">focused</a> | <code>true</code> | <code>BEMJSON</code>, <code>JS</code> | The block is in focus. |
 | <a href="#theme">theme</a> | <code>islands</code> | <code>BEMJSON</code> | A custom design. |
 | <a href="#size">size</a> | <code>'s'</code>, <code>'m'</code>, <code>'l'</code>, <code>'xl'</code> | <code>BEMJSON</code> | The size of the link. Use sizes only for links when <a href="#themes">theme modifier is set to islands</a>.|
```

Comment:
Use sizes for links only when the <a href="#themes">theme modifier is set to islands</a>.